src/front/app/reducer/app.spec.js

```diff
@@ -3,13 +3,15 @@ import appReducer from './app';
 describe('reducer/app', () => {
 
     let sut;
+    let challengeTopics;
 
     beforeEach(() => {
+        challengeTopics = [];
         sut = appReducer();
     });
 
-    it('should be empty object', () => {
-        sut.should.eqls({});
+    it('should contain challengeTopics reducer', () => {
+        sut.should.eqls({ challengeTopics });
```

Comment:
Will this test fail when more reducers are added to the application? Maybe it is better to use smth like `contains` matcher?